docs/community/working-group.md

```diff
@@ -195,3 +197,66 @@ Working group members are individuals and in many cases volunteers.
 There may be situations where a working group member might exhibit behavior that is objectionable,
 exceed the authority defined in this document or violate the [Code of Conduct](../../CODE_OF_CONDUCT.md).
 We recommend to report such issues by using the [Report Content](https://docs.github.com/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam) mechanism to bring it to the attention of the maintainers to review.
+
+## Language Working Group Process
+
+The PowerShell Language WG will follow the [C#](https://github.com/dotnet/csharplang) model for language enhancements.
+For _new feature proposals_, please create a [discussion](https://github.com/orgs/PowerShell/discussions/new?category=enhancements),
+and only submit a pull request if invited to do so by a member of the Language WG (a "champion").
+
+### Discussions
+
+Debate pertaining to language features takes place in the form of [Discussions](https://github.com/PowerShell/discussions) in this repo.
+If you want to suggest a feature, discuss current design notes or proposals, etc.,
+please [open a new Discussion topic](https://github.com/orgs/PowerShell/discussions/new?category=enhancements),
+
+Discussions that are short and stay on topic are much more likely to be read.
+If you leave comment number fifty, chances are that only a few people will read it.
+To make discussions easier to navigate and benefit from, please observe a few rules of thumb:
+
+- Discussion should be relevant to PowerShell language design. If they are not, they will be summarily closed.
+- Choose a descriptive topic that clearly communicates the scope of discussion.
+- Stick to the topic of the discussion. If a comment is tangential, or goes into detail on a subtopic, start a new discussion and link back.
+- Is your comment useful for others to read, or can it be adequately expressed with an emoji reaction to an existing comment?
```

Comment:
```suggestion
- Consider expressing support for a proposal with a reaction emoji rather than a comment, if the comment would otherwise not add more relevant information
```